charts/cluster-addons/values.yaml

```diff
@@ -89,25 +88,32 @@ openstack:
       name: openstack-cinder-csi
       version: 2.28.1
     values: {}
-    # Variables affecting the definition of the storage class
-    storageClass:
-      # Indicates if the storage class should be enabled
-      enabled: true
-      # The name of the storage class
-      name: csi-cinder
-      # Indicates if the storage class should be annotated as the default storage class
-      isDefault: true
-      # The reclaim policy for the storage class
-      reclaimPolicy: Delete
-      # Indicates if volume expansion is allowed
-      allowVolumeExpansion: true
-      # The Cinder availability zone to use for volumes provisioned by the storage class
-      availabilityZone: nova
-      # The Cinder volume type to use for volumes provisioned by the storage class
-      # If not given, the default volume type will be used
-      volumeType:
-      # The allowed topologies for the storage class
-      allowedTopologies:
+    defaultStorageClass: {}
+      # name: csi-cinder-fast
+      # # The reclaim policy for the storage class
+      # reclaimPolicy: Delete
+      # # Indicates if volume expansion is allowed
+      # allowVolumeExpansion: true
+      # # The Cinder availability zone to use for volumes provisioned by the storage class
+      # availabilityZone: nova
+      # # The Cinder volume type to use for volumes provisioned by the storage class
+      # # If not given, the default volume type will be used
+      # volumeType: fast
+      # # The allowed topologies for the storage class
+      # allowedTopologies:
```

Comment:
Please keep the same defaults as we had previously so as not to break existing installations that just use the defaults.